client/src/TenderClient.js

```diff
@@ -0,0 +1,78 @@
+import axios from "axios";
```

Comment:
If we are going to use axios, we need to add it as a dependency in `package.json`